src/jit/importer.cpp

```diff
@@ -13351,7 +13351,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         }
                     }
 
-                    if (prefixFlags & PREFIX_UNALIGNED)
```

Comment:
I don't understand the code that follow this, so I would just leave this line alone.

But you also might want to correct the broken indention here...